docs/en/observability/cloud-monitoring/aws/ingest-aws-beats.asciidoc

```diff
@@ -0,0 +1,39 @@
+[[ingest-aws-beats]]
+== Ingest with Beats
+
+++++
+<titleabbrev>Ingest with Beats</titleabbrev>
+++++
+
+Beats are lightweight data shippers that send operational data to Elasticsearch. Elastic provides separate Beats for different types of data, such as logs, metrics, and uptime. Depending on what data you want to collect, you may need to install multiple shippers on a single host.
+
+[discrete]
+[[aws-beats-data-streams]]
+=== What type of data can I collect
+
+With Beats you can collect two types of data: logs and metrics. Use https://www.elastic.co/guide/en/beats/filebeat/8.12/filebeat-overview.html[Filebeat module] for AWS logs and https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-overview.html[Metricbeat module] for AWS metrics.
+
+[discrete]
+[[aws-beats-prerequisites]]
+=== Prerequisites
+
+* An account on Elastic Cloud and a deployed stack on AWS. For instructions, refer to Installing the Elastic Stack.
```

Comment:
We should turn all of these "Installing the Elastic Stack" into links. At this point if the user hasn't set up the Elastic Stack, they need to know where to do so.

Reply:
This page is no longer included in the latest structure.